ext-apiserver/cloud/google/pods/apiserver.yaml

```diff
@@ -0,0 +1,242 @@
+
```

Comment:
This is not a pod. Perhaps we should put rename the directory.

Reply:
Maybe "deploy" or "config"?

Reply:
No to deploy. Maybe config or bundles? Naming is so hard. Just not pods.